numpy/lib/tests/test_arraypad.py

```diff
@@ -235,11 +235,11 @@ def test_check_minimum_1(self):
         a = np.arange(100)
         a = np.pad(a, (25, 20), 'minimum')
         b = np.array(
-            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-             0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-             0, 0, 0, 0, 0,
+            [ 0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
```

Comment:
I'm curious how the alignment is maintained.

Reply:
Manually if I really correctly. I know you would like a formatter able to properly format numerical arrays, but alas we're not there yet.